staff/clara-rufi/Precurs2019/PasapalabraTimer/Pasapalabra.js

```diff
@@ -0,0 +1,413 @@
+    { letter: "x", answer: "botox", status: 0, question: "CONTIENE LA X. Toxina bacteriana utilizada en cirujía estética"},
+    { letter: "y", answer: "peyote", status: 0, question: "CONTIENE LA Y. Pequeño cáctus conocido por sus alcaloides psicoactivos utilizado de forma ritual y medicinal por indígenas americanos"},
+    { letter: "z", answer: "zen", status: 0, question: "CON LA Z. Escuela de budismo que busca la experiencia de la sabiduría más allá del discurso racional"},
+]
+ 
+
+
+
+
+
+// Pasapalabra
+
+nameUser = []
+    document.getElementById('start').style="display:none;"
+    document.getElementById('letters').style="display:none;"
+    document.getElementById('showQuestion').style="display:none;"
+    document.getElementById('userAnswer').style="display:none;"
+    document.getElementById('acceptar').style="display:none;"
+    document.getElementsByClassName('letter').style="display:none;"
+    document.getElementById('Okrepeat').style="display:none;"//pq no es mostri
+    document.getElementById('repeatPasapalabra').style="display:none;"
+    document.getElementById('showresults').style="display:none;"
+    document.getElementById('results').style="display:none;"
+    document.getElementById("end").style="display:none;" 
+    document.getElementById("questionPlayAgain").style="display:none;"
+    document.getElementById("playAgainYes").style="display:none;"
+    document.getElementById("playAgainNo").style="display:none;"
+    document.getElementById("goodBye").style="display:none;"
+    document.getElementById("time").style="display:none;"
+    document.getElementById("tiempo").style="display:none;"
+    document.getElementById("minuts").style="display:none;"
+    document.getElementById("timeFinished").style="display:none;"
+
+function login(){
+
+var name = document.getElementById("nameUser").value
+document.getElementById("welcome").value = ""
+    if (name.length >=2){
+        nameUser.push(name)
+        console.log(nameUser)
+        document.getElementById("welcome").value += "Bienvenido " + nameUser + " !!"
+        document.getElementById('start').style="display:block;"
+    }else{
+        document.getElementById("welcome").value += "Nombre válido mínimo 2 carácteres"
+        document.getElementById("nameUser").value = ""
+    }
+    
+}
+
+
+n = 0
+function start(){
+    document.getElementById('letters').style="display:block;"
+    document.getElementById('showQuestion').style="display:block;"
+    document.getElementById('userAnswer').style="display:block;"
+    document.getElementById('loginUser').style="display:none;" 
+    document.getElementById('acceptar').style="display:block;"
+    document.getElementById('end').style="display:block;"
+    document.getElementById("time").style="display:block;"
+    document.getElementById("tiempo").style="display:block;"
+    document.getElementById("minuts").style="display:block;"
+    timer();
```

Comment:
In referenece to this [comment](https://github.com/manuelbarzi/skylab-bootcamp-201901/pull/42/files#r246115218), in this case you should call startTimer function.

```javascript
startTimer(180, document.querySelector('#time'));
```

As you can see, I'm pass the total seconds (180 = 3 minuts) and the querySelector.